source/AndroidResolver/src/FileMatchPattern.cs

```diff
@@ -0,0 +1,3 @@
+namespace GooglePlayServices {
+	public delegate bool FileMatchPattern(string filename);
```

Comment:
I am not entirely sure the benefit to introduce this delegate and the change in `PlayServicesResolver.cs`. I do not think there would be any performance gain, is there?

Reply:
I agree, removed it!